src/components/Create/BudgetForm.tsx

```diff
@@ -93,7 +107,11 @@ export default function BudgetForm({
             <Switch
               checked={showFundingFields}
               onChange={checked => {
-                setTarget(checked ? '10000' : maxIntStr || '0')
+                let target = checked ? '10000' : maxIntStr || '0'
```

Comment:
**nitpick (non-blocking)**: think we can make this a const:

```suggestion
                const target = checked ? '10000' : maxIntStr || '0'
```

Also, WDYT about making `'10000'` a constant at the top of the file, something like
```js
const DEFAULT_TARGET = '10000';
```
```